cns/simulator/simulator.go

```diff
@@ -78,6 +78,7 @@ func (m *CnsVolumeManager) CnsCreateVolume(ctx context.Context, req *cnstypes.Cn
 					BackingObjectDetails:         createSpec.BackingObjectDetails.(cnstypes.BaseCnsBackingObjectDetails).GetCnsBackingObjectDetails(),
 					ComplianceStatus:             "Simulator Compliance Status",
 					DatastoreAccessibilityStatus: "Simulator Datastore Accessibility Status",
+					HealthStatus:                 "green",
```

Comment:
I think you can use - https://github.com/vmware/govmomi/blob/master/pbm/types/enum.go#L112

```
PbmHealthStatusForEntityGreen   = PbmHealthStatusForEntity("green")
```

Reply:
Fixed.